optuna/study.py

```diff
@@ -623,9 +641,11 @@ def _run_trial(
     def _log_completed_trial(self, trial_number, value):
         # type: (int, float) -> None
 
-        _logger.info('Finished trial#{} resulted in value: {}. '
-                     'Current best value is {} with parameters: {}.'.format(
-                         trial_number, value, self.best_value, self.best_params))
+        msg = (
+            'Finished trial#{} resulted in value: {:.03f}. '
```

Comment:
Though specifying `:.03f` seems convenient for most of the users, I'm a bit worried that some tasks (e.g., [The Slice Localization dataset of HPOBench](https://user-images.githubusercontent.com/181413/70700061-73a20600-1d0d-11ea-8040-545aa267b710.png)) might have very small objective values that always be shown as `0.000` in this log message (the scale of the objective values depends on the target task).
I think that this change isn't essentially required by this progress bar PR (maybe), so we can address and discuss the improvement of this log message in another PR.

Reply:
This change derives from the obsolete/removed progress bar to display logging messages. Thus I can revert this change w/o problems.